src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -567,23 +565,31 @@ void *frontendservicesesion(std::string masterIP, int connFd, SQLiteDBInterface
                 continue;
             }
 
-            // We get the name and the path to graph as a pair separated by |.
+            // We get the name .
             char topic_name[FRONTEND_DATA_LENGTH];
             bzero(topic_name, FRONTEND_DATA_LENGTH + 1);
-
             read(connFd, topic_name, FRONTEND_DATA_LENGTH);
 
+            string con_message = "successfully get kafka topic";
+            int con_result_wr = write(connFd, con_message.c_str(), con_message.length());
+            if (con_result_wr < 0) {
+                frontend_logger.log("Error writing to socket", "error");
+                loop = true;
+                continue;
+            }
+
+            Utils utils;
             string topic_name_s(topic_name);
+            frontend_logger.log("sandaruwan " , "info");
             topic_name_s = utils.trim_copy(topic_name_s, " \f\n\r\t\v");
-
-            //std::thread streamingThread(KafkaConnector::startStream,topic_name_s, workerClients, streamsState);
-            //TODO(miyurud):Temporarily commenting this line to enable building the project. Asked tmkasun to provide a
-            // permanent fix later when he is available.
-            //streamsState->insert(topic_name_s, false);
+            kstream.Subscribe(topic_name_s);
+            frontend_logger.log("Start listening to " + topic_name_s, "info");
+            th1 = thread(listen_to_kafka_topic, std::ref(kstream),std::ref(graphPartitioner),std::ref(workerClients));
 
         } else if (line.compare(STOP_STREAM_KAFKA) == 0) {
             frontend_logger.log("Start serving `" + STOP_STREAM_KAFKA + "` command", "info");
-            string message = "send kafka topic name";
+            kstream.Unsubscribe();
+            string message = "Successfully stop kafka stream";
```

Comment:
We need to mention which Kafka stream we have stopped.

Reply:
Ok sir I mentioned it